src/main/java/com/mulesoft/connector/github/api/Domain/Issue.java

```diff
@@ -0,0 +1,105 @@
+package com.mulesoft.connector.github.api.Domain;
+
+import java.util.ArrayList;
+
+public class Issue {
+    private String url;
+    private String repository_url;
+    private String labels_url;
```

Comment:
Respect Camel Case notation as convention in all attribute names. If you need to return in Snake Case format, you can use @JsonProperty